src/benchmarks/micro/corefx/System.Threading.Tasks.Dataflow/Perf.BroadcastBlock.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Threading.Tasks.Dataflow.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class BroadcastBlockPerfTests : ReceivablePropagatorPerfTests<BroadcastBlock<int>, int>
+    {
+        public override BroadcastBlock<int> CreateBlock() =>
+            new BroadcastBlock<int>(
+                i => i,
+                new ExecutionDataflowBlockOptions
+                {
+                    MaxDegreeOfParallelism = Environment.ProcessorCount
+                });
```

Comment:
BroadcastBlock takes DataflowBlockOptions, not ExecutionDataflowBlockOptions, meaning it pays no attention to MaxDegreeOfParallelism.

Reply:
I agree. Fixed.